packages/events-example/README.md

```diff
@@ -1,10 +1,11 @@
 # Nile Events Example #
 
-This example demonstrates how to use a Nile agent to synchronize your data 
+This example demonstrates how to create a Nile agent to synchronize your data 
```

Comment:
@scottydawg there are other references to Pulumi in the README.md.  IIUC the target customer is not using Pulumi, so would it be worth extracting the Pulumi text into its own distinct section of the README?  "If using Pulumi then ..." (use existing text).  "If not using Pulumi then..." (give instructions for the hooks)

Reply:
The target customer might not be using Pulumi, but it's pretty central to the example. Does that next sentence clarify it?

> Nile doesn't prescribe any particular deployment solution, but here we'll be using [Pulumi](https://app.pulumi.com/) to deploy objects into AWS.

Reply:
@scottydawg that helps.  For the developer who is *not* using Pulumi, are there instructions on what to do?